alvar_list:
  - id: 1
    x: -0.2
    y: 0.0
    z: 0.0
    yaw: 0.0